src/components/Queues.ts

```diff
@@ -119,7 +119,15 @@ export class Queues extends Component<
         }
     }
 
-    async permissions(): Promise<PolicyStatement[]> {
-        return Promise.resolve([]);
+    permissions(): PolicyStatement[] {
+        const configuration = this.getConfiguration() ?? {};
+
+        return Object.entries(configuration).map(([name]) => {
+            const cfId = formatCloudFormationId(`${name}`);
+
+            return new PolicyStatement("sqs:SendMessage", [
+                cfGetAtt(`${cfId}Queue`, "Arn"),
```

Comment:
This reference doesn't work: it will reference `XxxQueue`, whereas the CDK will generate a resource called `XxxQueueE3DFB796` (unique ID).

Reply:
Pourquoi tu utilises `cfGetAtt` ?